pages/common/bitcoin-cli.md

```diff
@@ -23,3 +23,15 @@
 - Export the wallet information to a text file:
 
 `bitcoin-cli dumpwallet "{{path/to/file}}"`
+
+- Get blockchain info:
+
+`bitcoin-cli getblockchaininfo`
+
+- Get network info:
```

Comment:
```suggestion
- Get blockchain information:

`bitcoin-cli getblockchaininfo`

- Get network information:
```

This would align with the above descriptions.